src/rgw/store/sfs/bucket.h

```diff
@@ -21,18 +21,18 @@
 #include "common/Formatter.h"
 #include "common/ceph_json.h"
 #include "rgw_sal.h"
-#include "store/simplefile/bucket_mgr.h"
-#include "store/simplefile/multipart.h"
+#include "store/sfs/bucket_mgr.h"
+#include "store/sfs/multipart.h"
 
 
 namespace rgw::sal {
 
-class SimpleFileStore;
-class SimpleFileObject;
+class SFStore;
+class SFSObject;
 
-class SimpleFileBucket : public Bucket {
+class SFSBucket : public Bucket {
  private:
-  SimpleFileStore *store;
+  SFStore *store;
   BucketMgrRef mgr;
   const std::filesystem::path path;
   RGWAccessControlPolicy acls;
```

Comment:
Also, here, but it's commented. I would maybe remove this line or change it if we want to keep it, so there are no traces of old `simplefile` naming